File: app/src/main/kotlin/org/translate/core/yandex/Translate.java
package org.translate.core.yandex;

import com.google.gson.annotations.SerializedName;

import java.util.Arrays;

class Translate {
    @SerializedName("text")
    private String text;

    @SerializedName("pos")
    private String pos;

    @SerializedName("mean")
    private Mean[] means;

    @Override
    public String toString() {
        return text + " {" + pos + "}" + (means == null ? "" :
                " : " + Arrays.toString(means));
    }
}

File: app/src/main/kotlin/org/book2words/translate/yandex/Translate.kt
package org.book2words.translate.yandex

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Translate {
    @SerializedName("text")
    var text: String = ""

    @SerializedName("pos")
    var pos: String = ""

    //SerializedName("mean")
    @Expose
    var means: Array<Mean>? = null

    //SerializedName("syn")
    @Expose
    var syns: Array<Syn>? = null

    //SerializedName("ex")
    @Expose
    var examples: Array<Example>? = null
}
